spec/factories/users.rb

```diff
@@ -28,6 +32,12 @@
       end
     end
 
+    trait :developer do
```

Comment:
actually, developer role is assigned by default, via callback in the model.
So you can leave this trait as empty.

Reply:
deleted